azure-pipelines.yml

```diff
@@ -3,19 +3,67 @@
 
 # Build STL targeting x86, x64, arm, arm64
 
-jobs:
-- template: azure-devops/run-build.yml
-  parameters:
-    targetPlatform: x86
+stages:
+  - stage: Code_Format_Validation
+    displayName: "Code Format Validation"
+    jobs:
+      - job: Run_Code_Format_Validation
+        timeoutInMinutes: 90
+        displayName: "Run Code Format Validation"
+        pool:
+          name: StlBuild-2020-03-28
 
-- template: azure-devops/run-build.yml
-  parameters:
-    targetPlatform: x64
+        variables:
+          vcpkgLocation: '$(Build.SourcesDirectory)/vcpkg'
+        steps:
+          - checkout: self
+            clean: true
+            submodules: true
+          - task: run-vcpkg@0

```

Comment:
Does this not need/want the `vcpkg` cache?

Reply:
Nope it doesn't. In fact I wish I didn't have to run vcpkg here at all. Once https://github.com/lukka/CppBuildTasks/issues/19 is resolved we won't need to do the vcpkg step. If https://github.com/lukka/CppBuildTasks/issues/33 is resolved first we can drop the unnecessary boost-build.